cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/join/JoinField.java

```diff
@@ -63,7 +74,7 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "JoinField{" +
+    return "Field{" +
```

Comment:
The class name is still the same, right? This should still be called `JoinField`

Reply:
Done.